### PR TITLE
use versions from grails-bom

### PR DESCRIPTION
**Pull request by jamesfredley**

### CODE REVIEW

File: gradle/buildsrc.libs.versions.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [versions]
-asciidoctor-gradle-jvm = '4.0.2'
-assetpipeline = '5.0.1'
+asciidoctor-gradle-jvm = '4.0.3'
+assetpipeline = '5.0.4'
```
Comment (jdaugherty) [inline gradle/buildsrc.libs.versions.toml]: assetpipeline is in the bom, right?  Do we need it here?  5.0.5 is the version we should use to avoid that JS error

Comment (matrei) [reply gradle/buildsrc.libs.versions.toml]: Actually `asset-pipeline` is only used in a test app, and we don't need the gradle plugin there. So, it can be removed here and in the build dependencies of that test app.

**Review by jdaugherty — APPROVED** [review]

File: gradle/buildsrc.libs.versions.toml (hunk 1 above)
Comment (jdaugherty) [reply gradle/buildsrc.libs.versions.toml]: We can merge this then.  And create a ticket to do this later?

File: gradle/dependency-updates.gradle [hunk 2]
```diff
@@ -6,25 +6,16 @@ def groovyVersion = project.rootProject
...
-                details.because "The dependency coordinates are changed in Apache Groovy 4, plus ensure version"
-            }
 
-            if (details.requested.group == "io.micronaut" && details.requested.name == "micronaut-inject-groovy") {
-                details.useVersion(micronautVersion)
-            }
-        }
-    }
+dependencies {
+    implementation platform("org.grails:grails-bom:$grailsVersion")
```
Comment (matrei) [inline gradle/dependency-updates.gradle]: Why is this needed? Should we not add the bom to the TOML-file instead?
See https://github.com/gpc/grails-mail/blob/5.0.x/gradle/libs.versions.toml for reference.

File: gradle/buildsrc.libs.versions.toml (hunk 1 above)
Comment (jamesfredley) [reply gradle/buildsrc.libs.versions.toml]: Good point.  Removed it entirely.

**Review by matrei — APPROVED** [review]
I can make a pass over this build after this is merged.

File: gradle/dependency-updates.gradle (hunk 2 above)
Comment (jamesfredley) [reply gradle/dependency-updates.gradle]: Just an oversight on my part, will put this in TOML for now.

Comment (jamesfredley) [reply gradle/dependency-updates.gradle]: That was the first change I made before remembering this was on TOML.

Comment (jamesfredley) [reply gradle/dependency-updates.gradle]: fixed
